includes/Helpers/Plugin.php

```diff
@@ -66,6 +66,10 @@ public static function get_data( $basename, $data, $mu = false ) {
 		$plugin['mu']           = $mu;
 		$plugin['auto_updates'] = ( ! $mu && self::does_it_autoupdate( $basename ) );
 
+		if ( strpos( $basename, 'jetpack' ) !== false ) {
```

Comment:
Wouldn't this string match on other Jetpack plugins? Given that Jetpack has the main plugin and other standalone plugins, I'd imagine that maybe we should do an exact match.